package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Patient dashboard microfrontend for the OpenMRS SPA",
   "main": "dist/openmrs-esm-patient-chart.js",
   "scripts": {
-    "start": "webpack-dev-server",
+    "start": "webpack-dev-server --host 0.0.0.0",
```

Comment:
this makes it possible to debug using a different device because the app is now broadcasted on the network rather than just localhost